pkg/artifactory/resource_artifactory_remote_repository_test.go

```diff
@@ -547,6 +547,8 @@ func TestAccRemoteRepository_nugetNew(t *testing.T) {
 func mkNewRemoteTestCase(repoType string, t *testing.T, extraFields map[string]interface{}) (*testing.T, resource.TestCase) {
 	_, fqrn, name := mkNames("terraform-remote-test-repo-full", fmt.Sprintf("artifactory_remote_%s_repository", repoType))
 
+	certificateAlias := "certificate" + randSeq(10)
```

Comment:
Is `randSeq()` necessary? Would `randomInt()` suffice?

Reply:
changed to randomInt